src/site/_includes/partials/head.njk

```diff
@@ -32,15 +32,34 @@
 <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
 <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
 <link rel="mask-icon" color="#0054ff" href="/images/safari-pinned-tab.svg">
-<script type="module" src="{{ helpers.hashForProd('/js/app.js') }}"></script>
+{% set scriptLoader %}
+function loadScript(url, type) {
+  const s = document.createElement('script');
+  s.src = url;
+  if (type) s.type = type;
+  if (type === 'module') {
+    const pre = document.createElement('link');
+    pre.rel = 'modulepreload';
+    pre.href = url;
+    document.head.appendChild(pre);
```

Comment:
nit: you can use `.append` instead of `.appendChild`. (If you're using arrow functions, you can definitely use the more modern append).